core/src/test/kotlin/org/stellar/anchor/sep12/Sep12ServiceTest.kt

```diff
@@ -327,6 +328,75 @@ class Sep12ServiceTest {
     assertEquals(TEST_ACCOUNT, mockPutRequest.account)
   }
 
+  @Test
+  fun `Test put customer request with transaction_id`() {
```

Comment:
nit: We can combine this and the above the test into a `ParameterizedTest` on `PutCustomerRequest` since the only difference between them is the `transaction_id`.

Reply:
Good idea.